src/pocketmine/tile/Sign.php

```diff
@@ -89,29 +86,24 @@ public function getLine(int $index) : string{
 		if($index < 0 or $index > 3){
 			throw new \InvalidArgumentException("Index must be in the range 0-3!");
 		}
-		return (string) $this->namedtag["Text" . ($index + 1)];
+		return $this->namedtag{"Text" . ($index + 1)}->getValue();
```

Comment:
I'm not talking about the cast. You're missing a `->`.

 The cast wouldn't be needed regardless (it's there to shut PhpStorm up because it can't determine the type of the value gotten from the tag).